app/helpers/translatable_form_helper.rb

```diff
@@ -47,7 +51,14 @@ def translatable_field(field_type, method, options = {})
               final_options = @template.merge_translatable_field_options(options, locale)
                                        .reverse_merge(label: label_without_locale)
 
-              @template.concat send(field_type, localized_attr_name, final_options)
+              if field_type == :cktext_area
+                @template.concat content_tag :div, send(field_type, localized_attr_name, final_options),
+                                             class: "js-globalize-attribute",
+                                             style: @template.display_translation?(locale),
+                                             data: { locale: locale }
+              else
+                @template.concat send(field_type, localized_attr_name, final_options)
+              end
```

Comment:
We've already merged #2914 which incorporated these changes, in case you'd like to rebase against master.